src/Analyzer.Cli/Loggers/SummaryLogger.cs

```diff
@@ -56,42 +65,48 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
         /// <summary>
         /// Outputs to console a summary of the errors and warnings logged
         /// </summary>
-        public void SummarizeLogs(bool showVerboseMessage)
+        public void SummarizeLogs()
         {
+            // Local function to describe a count by returning the number with the appropriate form
+            // (singular or plural - just adding 's') of its description
+            string DescNum(int count, string description) => $"{count} {description}{(count == 1 ? "" : "s")}";
```

Comment:
Can you rename this func to PluralizeDescrip?

Reply:
I almost just named it "Pluralize", but I decided it's a little bit of a misnomer, since it won't actually pluralize if the count is 1.  I would be fine with "Pluralize" if that's preferable to "Describe Number".

Reply:
I'm not a fan of "Describe Number." It confused me. I will let others share their input.